docs/StardustDocs/topics/parse.md

```diff
@@ -54,8 +74,13 @@ df.parse(options = ParserOptions(locale = Locale.CHINA, dateTimeFormatter = Date
 <dataFrame src="org.jetbrains.kotlinx.dataframe.samples.api.Modify.parseWithOptions.html"/>
 <!---END-->
 
+### Global Parser Options
+
 You can also set global parser options that will be used by default in [`read`](read.md), [`convert`](convert.md),
-and `parse` operations:
+and other `parse` operations.
+These can be seen as a global fallback for the `parserOptions` argument.
```

Comment:
I suppose we need to be simpler with "fallback" definition and give a short explanation what is this, how it influences on something or add some references who are not familiar with that

Reply:
alright, expanded a bit on it